content/enterprise_influxdb/v1.9/administration/configure/security/ldap.md

```diff
@@ -18,7 +18,7 @@ Configure InfluxDB Enterprise to use LDAP (Lightweight Directory Access Protocol
 - Synchronize InfluxDB and LDAP so each LDAP request doesn't need to be queried
 
 {{% note %}}
-To configure InfluxDB Enterprise to support LDAP, all users must be managed in the remote LDAP service.
+LDAP **requires** JWT authentication. To configure InfluxDB Enterprise to support LDAP, all users must be managed in the remote LDAP service.
```

Comment:
Is this JWT authentication with InfluxDB, or JWT auth with the LDAP service itself?

Reply:
with InfluxDB

Reply:
Gotcha. We should link "JWT authentication" to `/enterprise_influxdb/v1.9/administration/configure/security/authentication/#configure-authentication-using-jwt-tokens`

Reply:
Done!!